docs/webhooks.rst

```diff
@@ -26,10 +29,14 @@ project was not imported through a connected account, you may need to
 manually configure a webhook for your project.
 
 To manually set up a webhook, click **Add integration** on your project's
-**Integrations** admin dashboard page and select the integration type you'd like
+**Integrations** Admin dashboard page and select the integration type you'd like
 to add. After you have added the integration, you'll see a URL for the
-integration on the :ref:`integration detail page <webhooks:Webhook Integrations>`. Use this
-URL when setting up a new webhook with your provider -- these steps vary
+integration once you go to readthedocs.org/dashboard, log in, and then go to 
```

Comment:
I'm not sure if is the best to point to the users to that link and the log in step, I think if the users are reading this they already in their project's admin page (the above step already point to that).

Reply:
Agreed, good point. Revising.